corehq/apps/style/templates/style/includes/modal_report_issue.html

```diff
@@ -81,7 +81,11 @@ <h4 class="modal-title">{% trans "Report an Issue with CommCare HQ" %}</h4>
                                           class="form-control"
                                           id="bug-report-project-description"
                                           rows="3"
-                                          placeholder="{% trans "Please provide a short description of your project." %}">{{ request.project.project_description }}</textarea>
+                                          placeholder="{% trans 'Please provide a short description of your organization and the work it is doing.' %}">{% spaceless %}
+                                          {% if request.project.project_description %}
```

Comment:
What's the `spaceless` for? django docs say that it's for removing space in between tags, but I don't think there should be any tags in the project description right?